pkg/cmd/pkg/push.go

```diff
@@ -27,12 +27,12 @@ func NewCmdPackagePush(f *factory.Factory) *cobra.Command {
 	var web bool
 
 	cmd := cobra.Command{
-		Use:   "push registry-name {path/to/file | --stdin-file-name filename -}",
-		Short: "Push a new package to Buildkite Packages",
+		Use:   "push registry-slug {path/to/file | --stdin-file-name filename -}",
+		Short: "Push a new package to a Buildkite registry",
 		Long: heredoc.Doc(`
-			Push a new package to Buildkite Packages. The package can be passed as a path to a file in the second positional argument,
-			or via stdin. If passed via stdin, the filename must be provided with the --stdin-file-name flag, as Buildkite
-			Packages requires a filename for the package.`),
+			Push a new package to a Buildkite registry. The package can be passed as a path to a file in the second positional argument,
```

Comment:
A "Buildkite registry" is more accurate and less cumbersome than a "Buildkite Package Registries registry".